src/main/java/com/agnieszkapawska/flashcards/domain/dtos/FlashcardDto.java

```diff
@@ -1,6 +1,6 @@
 package com.agnieszkapawska.flashcards.domain.dtos;
 
-import com.agnieszkapawska.flashcards.domain.models.QuestionTags;
+import com.agnieszkapawska.flashcards.domain.models.QuestionTag;
```

Comment:
What would you say about renaming this to Tag? I think in this project there will be no other kind of tags so we can use shorter name.